src/pages/FindWorkflow/FindWorkflowModal.test.js

```diff
@@ -89,6 +89,6 @@ describe('FindWorkflowModal', () => {
     fireEvent.click(dockstoreSubHeader)
 
     const dockstoreButton = screen.getByText('Go to Dockstore')
-    expect(dockstoreButton).toHaveAttribute('href', 'https://staging.dockstore.org/search?descriptorType=WDL&entryType=workflows&searchMode=files')
+    expect(dockstoreButton).toHaveAttribute('href', 'https://staging.dockstore.org/search?_type=workflow&descriptorType=WDL&searchMode=files')
```

Comment:
Not sure how this affects the opening of right Dockstore library, but I made this same as what we do in Terra UI.